src/test/java/com/fasttime/domain/resume/unit/service/ResumeServiceTest.java

```diff
@@ -0,0 +1,198 @@
+import com.fasttime.domain.resume.exception.ResumeNotFoundException;
+import com.fasttime.domain.resume.repository.ResumeRepository;
+import com.fasttime.domain.resume.service.ResumeService;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Optional;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@Slf4j
+@ExtendWith(MockitoExtension.class)
+class ResumeServiceTest {
+
+    public static final String MOCK_RESUME_TITLE = "Resume test1";
+    public static final String MOCK_RESUME_CONTENT = "# This is Resume";
+    public static final long MOCK_RESUME_ID = 1L;
+    @Mock
+    private MemberService memberService;
+    @InjectMocks
+    private ResumeService resumeService;
+    @Mock
+    private ResumeRepository resumeRepository;
+
+    @DisplayName("createResume()는")
+    @Nested
+    class Context_createResume {
+
+        @DisplayName("자기소개서를 DB에 성공적으로 저장한다.")
+        @Test
+        void _willSuccess() {
+            // given
+            Member member = Member.builder().id(1L).nickname("testName").build();
+            ResumeRequestDto requestDto = new ResumeRequestDto(MOCK_RESUME_TITLE,
+                    MOCK_RESUME_CONTENT);
+            Resume mockResume = createMockResume(member);
+
+            given(memberService.getMember(1L)).willReturn(member);
+            given(resumeRepository.save(any(Resume.class))).willReturn(mockResume);
+
+            // when
+            ResumeResponseDto response = resumeService.createResume(requestDto, 1L);
+
+            // then
+            assertThat(response).extracting("id", "title", "content", "writer", "likeCount", "viewCount")
+                    .containsExactly(1L, MOCK_RESUME_TITLE, MOCK_RESUME_CONTENT, "testName", 0, 0);
+        }
+
+    }
+
+    @DisplayName("updateResume()는")
+    @Nested
+    class Context_updateResume {
+
+        @DisplayName("자기소개서를 성공적으로 편집한다.")
+        @Test
+        void _willSuccess() {
+            // given
+            Member member = Member.builder().id(1L).nickname("testName").build();
+            Resume mockResume = createMockResume(member);
+
+            String updatedTitle = "updated title";
+            String updatedContent = "updated content";
+            ResumeUpdateServiceRequest updateRequest = new ResumeUpdateServiceRequest(
+                    MOCK_RESUME_ID, 1L,
+                    updatedTitle, updatedContent);
+
+            given(memberService.getMember(1L)).willReturn(member);
+            given(resumeRepository.findById(MOCK_RESUME_ID)).willReturn(Optional.of(mockResume));
+
+            // when
+            ResumeResponseDto response = resumeService.updateResume(updateRequest);
+            // then
```

Comment:
별 거 아니지만 공백 라인 하나가 누락된 것 같습니다~
```suggestion
            ResumeResponseDto response = resumeService.updateResume(updateRequest);

            // then
```